src/components/POSInput/types.ts

```diff
@@ -0,0 +1,7 @@
+import { OutlinedTextFieldProps } from '@material-ui/core/TextField';
+
+export interface POSInputProps extends Omit<OutlinedTextFieldProps, 'variant'> {
+  placeholder: string;
```

Comment:
these all exist in `OutlinedTextFieldProps`

So, you only need to Omit `variant` from `OutlinedTextFieldProps` and your good to go
```suggestionimport { OutlinedTextFieldProps } from '@material-ui/core/TextField';

export type POSInputProps = Omit<OutlinedTextFieldProps, 'variant'>;

```